packages/client.yml

```
@@ -71,6 +66,46 @@ switch:
+    optimistic: true
+    assumed_state: true # we can't read the state 
+    restore_mode: RESTORE_DEFAULT_OFF
+    turn_on_action:
+      - lambda: id(tesla_ble_vehicle_id)->sendCarServerVehicleActionMessage(SET_HVAC_SWITCH, 1);
+    turn_off_action:
+      - lambda: id(tesla_ble_vehicle_id)->sendCarServerVehicleActionMessage(SET_HVAC_SWITCH, 0);
+
+  - platform: template
+    name: "Steering Heater switch"
+    optimistic: true
+    assumed_state: true # we can't read the state 
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)